recipe/meta.yaml

```diff
@@ -1,61 +1,55 @@
-{% set version = "4.4.1" %}
+{% set version = "7.0.0" %}
 
 package:
   name: docker-py
   version: {{ version }}
 
 source:
   url: https://github.com/docker/docker-py/archive/{{ version }}.tar.gz
-  sha256: e28b9033c165e622562782691234bd1ac2da13d59bc60b389ecb9b7a3d297d43
+  sha256: 6a82c1630219a035c1717cf11082e97d109689c5e7f2da4aaabea4bb85a74faa
```

Comment:
This should fix the versioning issue because conda-forge built v7.0.0, see [here](https://github.com/conda-forge/docker-py-feedstock/blob/main/recipe/meta.yaml):
```
  url: https://pypi.io/packages/source/d/docker/docker-{{ version }}.tar.gz
  sha256: 323736fb92cd9418fc5e7133bc953e11a9da04f4483f828b527db553f1e7e5a3
```